assets/js/xrp-ledger-toml-checker.js

```diff
@@ -1,5 +1,7 @@
 const TOML_PATH = "/.well-known/xrp-ledger.toml"
 const TIPS = '<p>Check if the file is actually hosted at the URL above, check your server\'s HTTPS settings and certificate, and make sure your server provides the required <a href="xrp-ledger-toml.html#cors-setup">CORS header.</a></p>'
+const TIPS_1 = '<p>Make sure you are entering a valid XRP address.</p>'
+const TIPS_2 = '<p>Make sure the wallet address has the domain field.</p>'
```

Comment:
Suggested terminology:

```suggestion
const TIPS_2 = '<p>Make sure the account has the Domain field set.</p>'
```